library/src/google-maps-provider.tsx

```diff
@@ -29,6 +29,13 @@ export interface GoogleMapsContextType {
   map?: google.maps.Map;
 }
 
+// Declare global maps callback function
+declare global {
+  interface Window {
+    mapsCallback: () => void;
+  }
+}
+
```

Comment:
Move to [/library/src/types/index.d.ts](https://github.com/ubilabs/google-maps-react-hooks/blob/develop/library/src/types/index.d.ts)